reco_utils/recommender/deeprec/models/base_model.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, hparams, iterator_creator, graph=None, seed=None):
             seed (int): Random seed.
         """
         self.seed = seed
-        tf.set_random_seed(seed)
+        tf.compat.v1.set_random_seed(seed)
```

Comment:
I we don't use tf v2 in this file,
we can simply do:
```
import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()
```
and use `tf.`

See [tf official doc](https://www.tensorflow.org/guide/migrate)

Reply:
this is to avoid the warnings, also in the future, if we decide to use tf2, this will work out of the box